### PR TITLE
Adding pre-commit files and cleaned up files

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
-> Use pre-commit instead of flake8 for code-quality checks. Pre-commit provides more hooks for quality checking and it also helps to auto-correct the Code-quality errors.

### Fixes
Easier for users to contribute and get correct formatting

### CODE REVIEW

File: cerberus/kubernetes/client.py [hunk 1]
```diff
@@ -45,8 +46,11 @@ def list_pods(namespace):
     try:
         ret = cli.list_namespaced_pod(namespace, pretty=True)
     except ApiException as e:
-        logging.error("Exception when calling \
-                       CoreV1Api->list_namespaced_pod: %s\n" % e)
+        logging.error(
+            "Exception when calling \
+                       CoreV1Api->list_namespaced_pod: %s\n"
+            % e
```
Comment (chaitanyaenr) [inline cerberus/kubernetes/client.py:52]: @paigerube14 The spacing here looks a little weird, thoughts?

Comment (paigerube14) [reply cerberus/kubernetes/client.py:52]: yes very weird. Every time I run the pre-commit run --all-files it edits back to that. Do you know what flake rule might be causing that. Trying to investigate

**Comment by paigerube14** [comment]
I'm not sure how to fix the last error in the CI: `./start_cerberus.py:110:69: W605 invalid escape sequence '\['`
I am not seeing it locally when running `pre-commit run --all-files`
Any help would be greatly appreciated

File: cerberus/prometheus/client.py [hunk 2]
```diff
@@ -2,25 +2,26 @@
...
 
 
 # Initialize the client
 def initialize_prom_client(distribution, prometheus_url, prometheus_bearer_token):
     global prom_cli
     if distribution == "openshift" and not prometheus_url:
-        url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa
+        url = runcommand.invoke(
+            r"""oc get routes -n openshift-monitoring -o=jsonpath='{
+            .items[?(@.metadata.name=="prometheus-k8s")].spec.host}'"""
```
Comment (amitsagtani97) [inline cerberus/prometheus/client.py:15]: keep the url without newline character

url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa

**Review by amitsagtani97 — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -21,9 +21,9 @@ jobs:
       - name: flake8 Lint
```
Comment (amitsagtani97) [inline .github/workflows/ci.yml:21]: We can remove flake8 from here..as we are replacing it with pre-commit.
This file can be updated similar to this - https://github.com/cloud-bulldozer/kraken/blob/master/.github/workflows/ci.ym

**Review by amitsagtani97 — CHANGES_REQUESTED** [review]

File: config/config.yaml [hunk 4]
```diff
@@ -43,7 +43,7 @@ cerberus:
 
 tunings:
     iterations: 5                                        # Iterations to loop before stopping the watch, it will be replaced with infinity when the daemon mode is enabled
-    sleep_time: 60                                       # Sleep duration between each iteration
+    sleep_time: 30                                       # Sleep duration between each iteration
```
Comment (amitsagtani97) [inline config/config.yaml:46]: @paigerube14 hey, wanted to make sure that this change in sleep time is updated by you? Not by the pre-commit.

Other things LGTM!

**Review by amitsagtani97 — APPROVED** [review]

File: config/config.yaml (hunk 4 above)
Comment (paigerube14) [reply config/config.yaml]: Yes, sorry. That was me but definitely want to keep 60 seconds as the default. Fixed!

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**